src/server/util/readingsUtils.js

```diff
@@ -184,6 +184,29 @@ const unitDatakWh = [
         displayable: Unit.displayableType.NONE,
         preferredDisplay: false,
         note: 'special unit'
+    },
```

Comment:
The setup of this file should not be changed by a test. You should add the needed units and conversions for this test inside your test. For example, see completed test L12 for how OED would like this done.

Reply:
Thanks for the feedback! We will update the PR and use L12 as an example

Reply:
We updated the PR with your feedback :) Let us know if you have other feedback!